packages/components/src/components/modal/modal.scss

```diff
@@ -31,7 +31,7 @@
         &--is-vertical-top {
             top: $HEADER_HEIGHT;
             position: absolute;
-            @include mobile {
+            @include new-mobile {
```

Comment:
Can we have different names for it without `new` prefix? It doesn't make sense to use the time sensitive prefixes in the code. Either we use `mobile` or something more detailed to the changes we have inside.